### PR TITLE
bump(main/faust): 2.74.6

### PR DESCRIPTION
**Pull request by harieamjari**

### CODE REVIEW

**Review by TomJo2000 — APPROVED** [review]

File: packages/faust/build.sh [hunk 1]
```diff
@@ -1,15 +1,12 @@
...
-TERMUX_PKG_VERSION=2.70.3
-_FAUSTLIB_COMMIT=2a5b4ab36e5d47f7e56277b19f41892bcf6378f4
-TERMUX_PKG_SRCURL=(https://github.com/grame-cncm/faust/archive/refs/tags/${TERMUX_PKG_VERSION}.tar.gz
-		https://github.com/grame-cncm/faustlibraries/archive/${_FAUSTLIB_COMMIT}.zip)
-TERMUX_PKG_SHA256=(644484f95167fe63014eac3db410f50c58810289fea228a2221e07d27da50eec
-		be37b2d32b213996ca2b7ab12c6d282d254dc20a1792c796d26f6bd3bda8bdd6)
+TERMUX_PKG_VERSION=2.74.6
+TERMUX_PKG_SRCURL=https://github.com/grame-cncm/faust/releases/download/${TERMUX_PKG_VERSION}/faust-${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=b42a4ed2f8f2eae46e8c37b9dc3ac1bfbdaf0079b75f97ba62968f87a701bb02
 TERMUX_PKG_AUTO_UPDATE=false
```
Comment (TomJo2000) [inline packages/faust/build.sh:7]: We should be able to re-enable auto updates now since the `TERMUX_PKG_SRCURL` is no longer an array.

**Review by TomJo2000 — CHANGES_REQUESTED** [review]

**Comment by TomJo2000** [comment]
I'll send up a quick commit to reenable auto updates and then this looks good to merge.

**Comment by harieamjari** [comment]
Looks like both of us force pushed.

**Comment by harieamjari** [comment]
I didn't see your comment. You can try force pushing again.

**Comment by TomJo2000** [comment]
Yep I noticed when I tried to add some clarification on the Licenses.
I usually use `git push --force-with-lease=<branch name>` to avoid overwriting changes on a branch.
Sorry about that, should have given you more of a heads up.

**Comment by TomJo2000** [comment]
This should be in a consistent state now.
You can "accept" this state on your branch locally using 
`git -c pull.rebase=true pull`

I again apologize for not giving you more of a heads up,
but hopefully we didn't clobber anything.

**Review by TomJo2000 — APPROVED** [review]

**Comment by harieamjari** [comment]
Don't mind. When this happens, I kind of just `git --reset hard HEAD^ && git pull` on my local.

**Pull request merged**